control-plane-operator/controllers/hostedcontrolplane/assets/apiserver-haproxy/kube-apiserver-proxy.yaml

```diff
@@ -13,15 +13,29 @@ spec:
     priorityClassName: system-node-critical
     securityContext:
       runAsUser: 1001
+    resources:
+      limits:
+        cpu: 300m
+        memory: 512M
+      requests:
+        cpu: 13m
+        memory: 16M
```

Comment:
These seem pretty specific.  Where to they come from?

Reply:
It is basically from kubectl top pod of the haproxy pod after some time and then just doubling that value for requests. Limits are based on what we run ROKS with for the past couple years.

Reply:
so requests:
Spot metrics during typical work usage using kubectl top pods -n kube-system * 2 

limits:
Values IBM Cloud has used for the past 2 years in production and have worked out well.

Reply:
It just seems strange to me that the limits are roughly 20x-30x the requests.  What is the _actual_ usage we expect in operation?  What situation can result in the haproxy pod getting near the limits?

Reply:
Actual usage would be slightly lower than the requests value: (10m CPU 9M memory). The limits are there for cases where there are memory leaks or run away CPU usage. It's high to accommodate for customers that have many components that hit the APIServer frequently (CPU/Mem goes up as it's processing more requests at a given period of time). 
They are just a high value at least for now that should never be hit no matter how high the usage one customer has for components calling the APIServer through haproxy.

Reply:
Fair enough.  Just wanted to make sure the requests were really close to actually normal usage.

/approve
/lgtm